src/gui/optionsdialog.ui

```diff
@@ -3187,18 +3186,39 @@ Disable encryption: Only connect to peers without protocol encryption</string>
                    </property>
                   </widget>
                  </item>
-                 <item row="0" column="2">
-                  <spacer name="horizontalSpacer_6">
-                   <property name="orientation">
-                    <enum>Qt::Horizontal</enum>
+                 <item row="2" column="0">
+                  <widget class="QLabel" name="label_12">
+                   <property name="text">
+                    <string>Maximum number of articles per feed:</string>
                    </property>
-                   <property name="sizeHint" stdset="0">
-                    <size>
-                     <width>40</width>
-                     <height>20</height>
-                    </size>
+                  </widget>
+                 </item>
+                 <item row="0" column="0">
+                  <widget class="QLabel" name="label_111">
+                   <property name="text">
+                    <string>Feeds refresh interval:</string>
                    </property>
-                  </spacer>
+                  </widget>
+                 </item>
+                 <item row="1" column="0">
+                  <widget class="QLabel" name="label_rss_fetch_delay">
+                   <property name="text">
+                    <string>Delay between requests:</string>
```

Comment:
Wouldn't `Delay requests to same host` be more clear?

Reply:
Ended up renaming it to `Same host request delay` as its a bit shorter.
at 3e91435acd2b343a8ed5c1f48c0285c509ecc1a5